server/controllers/interviewController.js

```diff
@@ -48,10 +48,25 @@ const startInterview = async (req, res) => {
 
   res.json(interview)
 }
+
+const getInterviewsByUserId = async (req, res) => {
+  const email = req.user.email;
+  const interviews = (await User.findOne({ email }).populate('interviews')).interviews;
+  res.json(interviews);
+}
+
+const cancelInterviewById = async (req, res) => {
+  const { id } = req.params
+  const interviews = await Interview.findByIdAndDelete(id)
+  res.json(interviews);
+}
```

Comment:
I wonder if it would make more sense to just end an interview rather than cancel/ delete it if it's already been started? Can either person end/cancel the interview at any time without consent of the other person?